### ISSUE TITLE
Support for new ad placement locations on home page feed

### ISSUE THREAD

**Issue by jaw6** [forem/forem#19225]
## User Story (Who / What / Why)
We want to update `DisplayAd` to support new `placement_areas` and update server-rendered home-page views to include these new areas for signed-out users. (Signed-in home-page views are rendered client-side and will update later.) (There is [a closely-related spin-off issue](https://github.com/forem/forem/issues/19258), both can theoretically work in parallel.)

New `placement_areas`:
* [ ] `feed_first` - Before all home page posts
* [ ] `feed_second` - Between 2nd and 3rd posts in the feed
* [ ] `feed_third` - Between 7th and 8th posts in the feed
* [ ] these options should be available at `/admin/customization/display_ads/new`
* [ ] these options should appear in API documentation for `display_ads`
* [ ] remove hard-coded `sign_in_invitation` (aka CTA)
* [ ] remove [event](https://github.com/forem/forem/blob/09291b420515d265eca031b431a5937bc63c3bb3/app/javascript/packs/homePage.jsx#L135) as we won't be tracking clicks on this CTA. 

Considerations:
* `feed` billboards should take up the full width of the feed, styled to match (text sizes, etc.)
* The initial implementation is focused on the home page articles feed, but later iterations may add these billboards to tagged articles feeds as well.
* These types of billboards are not able to target `logged_in` users _for now_, but we don't need to codify this as a validation.
* `feed_` placements can split into as many PRs make sense
* do **not** remove Campaign HtmlVariants (some communities use it)
* :link: There's additional background discussion of these new placement areas in 📝 [this document](https://docs.google.com/document/d/1biJjAjLsz2EHSLntJKViKQjDIjb36SRdmdarmE9dAPU/edit#) and design work [in Figma](https://www.figma.com/file/9UfSjNYJs745w3Q8JCUqsL/Small-Wins---%E2%80%A8Core-Experience-%E2%9A%A1%EF%B8%8F?node-id=1623-7063&t=Qj7oeBVeo7LrkobH-4) and [Loom](https://www.loom.com/share/8e9a5cb821524e4890c8f6df43d8d1e9)

**Comment by jaw6** [comment]
There was initially discussion of handling `home hero` in a separate PR while using one issue for both, however, there are outstanding product/design questions for `home hero`, so I am moving it to [a separate issue](https://github.com/forem/forem/issues/19258) and updating the language here to be more specific to `feed` ads.

**Comment by mirie** [comment]
Will need to communicate externally that we're removing the hard-coded CTA (for any Forems that are using this). Also just a heads up to @noah-brinker that we are planning to remove this account signup CTA away and to not expect these events to come in once this is removed.

**Comment by rt4914** [comment]
@mirie I am interested in working on this.

**Comment by noah-brinker** [comment]
Is this the CTA in the top left corner for logged out visitors that will be going away permanently?
<img width="340" alt="Screen Shot 2023-03-29 at 2 36 50 PM" src="https://user-images.githubusercontent.com/103451151/228650572-e905951a-29ee-400a-ac23-e609c351223a.png">

**Comment by rt4914** [comment]
@noah-brinker The CTA you mentioned will not be removed. We show another CTA in home-feed at 5th position, instead of article. This will be removed.
<img width="600" alt="Screenshot 2023-03-30 at 12 23 12 PM" src="https://user-images.githubusercontent.com/9396084/228753827-b72ec845-acb1-4dd6-99af-7e0ee7b08de1.png">

**Comment by noah-brinker** [comment]
Got it, thanks!
